Makefile

```diff
@@ -107,12 +107,15 @@ docs: .docker-build-pull
 	${MAKE} build-ci
 
 build-ci: .docker-build-pull
-	${DC_CI} build --pull release
```

Comment:
the `--pull` here ensures we're using the latest version of the base image. Do we not want that?